src/components/DetailProductComments.tsx

```diff
@@ -68,13 +80,19 @@ function EmptyComments() {
   );
 }
 
-function DetailProductComments({ comments }) {
+interface Comments {
+  comments: CommentProps['comment'][];
+}
```

Comment:
위에서 Comment 인터페이스를 정의했으니 
```suggestion
interface Comments {
  comments: Comment[];
}
```
로 해주면 더 보기 좋을 것 같아요!